src/components/typography/index.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from "react"
```

Comment:
oh we did not setup prettier check in CI https://github.com/oasisprotocol/ui-library/pull/61. Anyway, you prob missing some IDE settings. Prettier should format everything for you when you edit a file